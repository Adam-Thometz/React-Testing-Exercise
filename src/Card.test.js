import React from "react";
import { render, fireEvent } from "@testing-library/react";
import Card from "./Card";

it('renders without crashing', () => {
  render(<Card />)
})

it('matches snapshot', () => {
  const {asFragment} = render(<Card caption="Hello!!" src="https://upload.wikimedia.org/wikipedia/commons/thumb/5/59/-127wiki.jpg/1920px--127wiki.jpg" currNum={1} totalNum={3} />)
  expect(asFragment()).toMatchSnapshot()
})