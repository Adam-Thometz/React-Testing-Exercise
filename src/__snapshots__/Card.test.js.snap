// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      Hello!!
    </h4>
    <img
      alt="Hello!!"
      class="Card-image"
      src="https://upload.wikimedia.org/wikipedia/commons/thumb/5/59/-127wiki.jpg/1920px--127wiki.jpg"
    />
    <small
      class="Card-small"
    >
      Image 1 of 3.
    </small>
  </div>
</DocumentFragment>
`;
